### ISSUE TITLE
Referencing android libraries from project.properties?

### ISSUE THREAD

**Issue by vkislicins** [robolectric/robolectric#1619]
I can't get the unit tests running with Robolectric. I think the problem is that Robolectric is not loading/parsing appcompat. As I understood from other issues I need to create project.properties to point at those libs, however I am not clear on 2 things:

1) Where do I put project.properties in my structure?
2) Where does Robolectric load those libs from, i.e. relatively to what am I constructing the path?

My structure is something like this:

```
-project

----build
----gradle
----libraries
----my-app-a
--------build
--------src
----my-app-b
--------build
--------src
```

My libraries get unpacked under specific app, so for example I have project/my-app-a/build/intermediates/exploded-aar/com.android.support/appcompat-v7/21.0.0

So what should I put in project.properties?

```
android.library.reference.1=???/build/intermediates/exploded-aar/com.android.support/appcompat-v7/21.0.0
android.library.reference.2=???/build/intermediates/exploded-aar/com.android.support/support-v4/21.0.0
```

And where should I put this file?

Error message that I'm getting:

```
java.lang.IllegalStateException: You need to use a Theme.AppCompat theme (or descendant) with this activity.
	at android.support.v7.app.ActionBarActivityDelegate.onCreate(ActionBarActivityDelegate.java:151)
	at android.support.v7.app.ActionBarActivityDelegateBase.onCreate(ActionBarActivityDelegateBase.java:138)
	at android.support.v7.app.ActionBarActivity.onCreate(ActionBarActivity.java:123)
```

**Comment by erd** [comment]
> Where do I put project.properties in my structure?

If you are building with Gradle, you do not need to create a project.properties because the build system will merge the resources of all the external libraries and your application's resources for you. project.properties should only be necessary for people using Maven (or other custom build systems).

The easiest thing for you is to create a custom test runner that specifies the merged resource directory for robolectric. A good example can be found here: https://github.com/nenick/AndroidStudioAndRobolectric/blob/master/app/src/test/java/com/example/myapplication/CustomRobolectricRunner.java. Additional discussion can be found here: https://github.com/robolectric/robolectric-gradle-plugin/issues/142.

I'm working on a custom test runner similar to the above that will be included in 3.0 and hopefully alleviate this issue.

**Comment by vkislicins** [comment]
That test runner example worked for me - thank you very much!!
